proto/account/service.proto

```diff
@@ -932,4 +941,50 @@ service AccountService {
       }
     };
   }
+  rpc CreateSearchFilter(CreateSearchFilterRequest)
+      returns (CreateSearchFilterResponse) {
+    option (google.api.http) = {
+      post: "/v1/account/create_search_filter"
+      body: "*"
+    };
+    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+      summary: "Create Search Filter",
+      description: "Create a search filter to use in the console.",
```

Comment:
```suggestion
      description: "Create a search filter. To call this API, you need an `EDITOR` role.",
```

Reply:
Usually, a viewer can only read data on the console. They are not allowed to create or update anything.
But since this is the user's settings, it should be fine to use Viewer for this case.